src/consensus/aft/orc_requests.h

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ds/thread_messaging.h"
+#include "kv/kv_types.h"
+#include "node/node_client.h"
+#include "node/rpc/serdes.h"
+
+#include <chrono>
+
+namespace ccf
+{
+  struct ObservedReconfigurationCommit
+  {
+    struct In
+    {
+      ccf::NodeId from;
+      kv::ReconfigurationId reconfiguration_id;
+    };
+
+    using Out = void;
+  };
+
+  DECLARE_JSON_TYPE(ObservedReconfigurationCommit::In)
+  DECLARE_JSON_REQUIRED_FIELDS(
+    ObservedReconfigurationCommit::In, from, reconfiguration_id)
+
+  inline bool submit_orc(
+    std::shared_ptr<ccf::NodeClient> client,
+    const ccf::NodeId& from,
+    kv::ReconfigurationId rid)
+  {
+    LOG_DEBUG_FMT("Configurations: submit ORC for #{} from {}", rid, from);
+
+    ObservedReconfigurationCommit::In ps = {from, rid};
+
+    http::Request request(fmt::format(
+      "/{}/{}", ccf::get_actor_prefix(ccf::ActorsType::nodes), "orc"));
+    request.set_header(
+      http::headers::CONTENT_TYPE, http::headervalues::contenttype::JSON);
+
+    auto body = serdes::pack(ps, serdes::Pack::Text);
+    request.set_body(&body);
+    return client->make_request(request);
+  }
+
+  struct AsyncORCTaskMsg
+  {
+    AsyncORCTaskMsg(
+      std::shared_ptr<ccf::NodeClient> client_,
+      const ccf::NodeId& from_,
+      kv::ReconfigurationId rid_) :
+      client(client_),
+      from(from_),
+      rid(rid_)
+    {}
+
+    std::shared_ptr<ccf::NodeClient> client;
+    ccf::NodeId from;
+    kv::ReconfigurationId rid;
+  };
+
+  inline void orc_cb(std::unique_ptr<threading::Tmsg<AsyncORCTaskMsg>> msg)
+  {
+    if (!submit_orc(msg->data.client, msg->data.from, msg->data.rid))
+    {
+      threading::ThreadMessaging::thread_messaging.add_task_after(
+        std::move(msg), std::chrono::milliseconds(250));
```

Comment:
why 250?

Reply:
Because it felt right! This is a re-schedule of a failed RPC (need to keep sending ORCs forever in general), but I didn't want to reschedule without any delay to avoid liveness issues.

Reply:
Ok! Can we have a constexpr ORC_RPC_RETRY_INTERVAL_MS, and use that here? I think it'd be more readable.